### ISSUE TITLE
Any way to prolong history?

### ISSUE THREAD

**Issue by cyberteaborg** [tapchat/tapchat#12]
Hello, thank you for your app. Just wanted to ask if there is any way to make channel's history longer? Could not find any config for tapchat on my server(

Also is it possible to add different per server encodings support?

This is my first issue on github, I hope I didn't do anything wrong)

**Comment by mpontes** [comment]
I really, really, really would like this as well. History available on TapChat is far too short for me, I would love to have the option to set it to a higher number.

EDIT: The obvious fix is to replace the hardcoded 50 in backlog_db with something else (I set it to 500 and it worked just fine for me), but obviously this bug is about being able to set a proper config value.

**Comment by sangh** [comment]
So I have a related problem and am using this space instead of opening another issue.  I am on a channel with about a thousand people that join and part at different times (has to do with when a push is happening), there is not much chatter on the channel, but when someone says something I want to see it.  This is normally hard as there are hundreds of join and part messages for each line someone says.  Now on a normal client I just turn off join/part notifications and it works fine.  On TapChat, however every time someone joins or parts it uses a "line" in the history which has the effect of whenever I open the channel on TapChat all I see is two lines, one listing the parts and the other listing the joins (the fact that multiple joins and parts and grouped into one line is fantastic, BTW).  Could the joins and parts be made to not use a whole line of history?  (In the meantime I'm going to change the value in backlog_db.)

**Comment by blunden** [comment]
Yes, the backlog should be configurable. I usually use several days worth of backlog in irssi.
